### PR TITLE
New names for NULL counties in Qatar

### LINKED ISSUE #1230: Six admin2 in Qatar with NULL name
https://spelunker.whosonfirst.org/search/?q=NULL&iso=qa

Update `wof:name` property, ditch the "NULL" name.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #1230.

Includes new `wof:name` and `name:eng_x_preferred` name values, sourced from the Qatari government's population and statistics map, here: https://www.mdps.gov.qa/en/knowledge/Publications/Population/Population2015.pdf

`NULL` values were not stored as alt names.

### CODE REVIEW

File: data/110/875/809/1/1108758091.geojson [hunk 1]
```diff
@@ -23,6 +23,9 @@
     "mps:longitude":51.508355,
     "mz:hierarchy_label":1,
     "mz:is_current":1,
+    "name:eng_x_preferred":[
+        "Al Thumama / Al Wukair / Al Mashaf"
```
Comment (nvkelso) [inline data/110/875/809/1/1108758091.geojson]: Really?

File: data/110/875/810/9/1108758109.geojson [hunk 2]
```diff
@@ -23,6 +23,9 @@
     "mps:longitude":51.276526,
     "mz:hierarchy_label":1,
     "mz:is_current":1,
+    "name:eng_x_preferred":[
+        "Al Ruwais / Madinat Al Shamal"
```
Comment (nvkelso) [inline data/110/875/810/9/1108758109.geojson]: Really?

File: data/110/875/809/1/1108758091.geojson (hunk 1 above)
Comment (stepps00) [reply data/110/875/809/1/1108758091.geojson]: According to the doc, many of the "zone" divisions have multiple names, separated by ` /  `.

File: data/110/875/810/9/1108758109.geojson (hunk 2 above)
Comment (stepps00) [reply data/110/875/810/9/1108758109.geojson]: See above.

File: data/110/875/809/1/1108758091.geojson (hunk 1 above)
Comment (nvkelso) [reply data/110/875/809/1/1108758091.geojson]: Okay... https://en.wikipedia.org/wiki/Al_Wakrah_(municipality)#Census_zones

**Comment by nvkelso** [comment]
Noting here these are default English and preferred English names (missing Arabic), but that's fine for this cut.

File: data/110/875/809/1/1108758091.geojson (hunk 1 above)
Comment (stepps00) [reply data/110/875/809/1/1108758091.geojson]: Noting that the admin2 names in Qatar are sometimes groupings of the smaller division names, hence these new names..

**Pull request merged**